Documentation/operations/troubleshooting.rst

```diff
@@ -238,25 +236,21 @@ to set the ``hubble.enabled=true`` value.
 To check if Hubble is enabled in your deployment, you may look for the
 following output in ``cilium status``:
 
-.. code:: bash
+.. code:: shell-session
 
-    $ cilium status
-    ...
-    Hubble:   Ok   Current/Max Flows: 2542/4096 (62.06%), Flows/s: 164.21   Metrics: Disabled
-    ...
+   $ cilium status
+   ...
+   Hubble:   Ok   Current/Max Flows: 2542/4096 (62.06%), Flows/s: 164.21   Metrics: Disabled
+   ...
 
 .. note::
-    Pods need to be managed by Cilium in order to be observable by Hubble.
-    See how to :ref:`ensure a pod is managed by Cilium<ensure_managed_pod>`
-    for more details.
+   Pods need to be managed by Cilium in order to be observable by Hubble.
+   See how to :ref:`ensure a pod is managed by Cilium<ensure_managed_pod>`
+   for more details.
 
 Observing flows with Hubble Relay
 =================================
 
-.. note::
-   **Hubble Relay** is beta software and as such is not yet considered
-   production ready.
-
 Hubble Relay is a service which allows to query multiple Hubble instances
 simultaneously and aggregate the results. As Hubble Relay relies on individual
 Hubble instances, Hubble needs to be enabled when deploying Cilium. In
```

Comment:
As Hubble is enabled by default in 1.9, this can be simplified a bit:

> Hubble Relay is a service which allows to query multiple Hubble instances simultaneously and aggregate the results. The Hubble service needs to be exposed on TCP port 4244 to allow Hubble Relay to connect to individual instances. This can be done via Helm values  or option ``--hubble-listen-address :4244`` on cilium-agent.

Also, the note below about security implications can be removed as the endpoint is now secured via mTLS by default.